data/governments.txt

```diff
@@ -546,6 +546,20 @@ government "Hai (Unfettered)"
 	"hostile hail" "hostile unfettered"
 	"hostile disabled hail" "hostile disabled unfettered"
 
+government "Hai (Unfettered Challenger)"
+	"display name" "True Hai (Glory Seeker)"
```

Comment:
```suggestion
	"display name" "True Hai (Combatant)"
```

I think "Glory Seeker" is a little odd. Also, I'm not sure about "True Hai" instead of Unfettered.

Reply:
I'd like another opinion on this, I do personally like glory seeker
They're not just combatants, they're duellists

Reply:
they're Unfettered Heshi now

Reply:
👍